src/transformers/models/detr/modeling_detr.py

```diff
@@ -1553,29 +1567,36 @@ def forward(
 
         ```python
         >>> from transformers import DetrFeatureExtractor, DetrForSegmentation
+        >>> from transformers.models.detr.feature_extraction_detr import rgb_to_id
+        >>> import torch
         >>> from PIL import Image
+        >>> import numpy
         >>> import requests
+        >>> import io
```

Comment:
This is not very important but importing low-level libraries first improves readability
```
import io
import requests
from PIL import Image
import torch
import numpy

from transformers import DetrFeatureExtractor, DetrForSegmentation
from transformers.models.detr.feature_extraction_detr import rgb_to_id
```